test/runtests.jl

```diff
@@ -19,28 +19,22 @@ indata, inhdr = readdlm(fname, ','; header = true)
 # Float32 has a large tolerance compared to Float64 and BigFloat, but here we
 # are more interested in making sure that the infrastructure works for different
 # floating types.
-@testset "Testing $F" for (F, TOL) in (
-    (Float32, 0.2),
+@testset "Testing $F" for (F, TOL) in ((Float32, 0.2),
     (Float64, 0.0001),
-    (BigFloat, 0.0001),
-)
-    for (insys, T) in (
-        ("icrs", ICRSCoords{F}),
+    (BigFloat, 0.0001),)
+    for (insys, T) in (("icrs", ICRSCoords{F}),
         ("fk5j2000", FK5Coords{2000,F}),
         ("fk5j1975", FK5Coords{1975,F}),
-        ("gal", GalCoords{F}),
-    )
+        ("gal", GalCoords{F}),)
         c_in = T[T(indata[i, 1], indata[i, 2]) for i = 1:size(indata, 1)]
         for c in c_in
             @test convert(T, c) == c
         end
 
-        for (outsys, S) in (
-            ("icrs", ICRSCoords{F}),
+        for (outsys, S) in (("icrs", ICRSCoords{F}),
             ("fk5j2000", FK5Coords{2000,F}),
             ("fk5j1975", FK5Coords{1975,F}),
-            ("gal", GalCoords{F}),
-        )
+            ("gal", GalCoords{F}),)
```

Comment:
Actually I think that the hanging parentheses here are better, they allow us to add more types or coordinates without touching other lines, useful to have a clean history.

Generally speaking, I'm not a big fan of touching lines without functional changes, these kinds of edits make using tools like `git blame` harder and the history more noisy and distracting.